packages/dds/tree/src/shared-tree/treeApiAlpha.ts

```diff
@@ -203,7 +203,12 @@ export const TreeAlpha: {
 		compressedData: JsonCompatible<IFluidHandle>,
 		options: { idCompressor?: IIdCompressor } & ICodecOptions,
 	): Unhydrated<TreeFieldFromImplicitField<TSchema>>;
-} = {
+}
+
+/**
+ * @sealed @alpha
+ */
+export const TreeAlpha = {
```

Comment:
Should this be typed as `TreeAlpha` as well?

Reply:
Yes. Good catch!

Reply:
FIxed.